Fix the build

CI is failing with:

```
ruby 3.1.1p18 (2022-02-18 revision 53f5fc4236) [x86_64-linux-musl]
Configuration file: /srv/jekyll/_config.yml
            Source: /srv/jekyll
       Destination: /srv/jekyll/_site
 Incremental build: disabled. Enable with --incremental
      Generating... 
  Conversion error: Jekyll::Converters::Scss encountered an error while converting 'css/main.scss':
                    end of file reached
                    ------------------------------------------------
      Jekyll 4.3.2   Please append `--trace` to the `build` command 
                     for any additional information or backtrace. 
                    ------------------------------------------------
/usr/gem/gems/sass-embedded-1.64.1-x86_64-linux-musl/lib/sass/embedded/varint.rb:20:in `readbyte': end of file reached (EOFError)
	from /usr/gem/gems/sass-embedded-1.64.1-x86_64-linux-musl/lib/sass/embedded/varint.rb:20:in `block in read'
	from /usr/gem/gems/sass-embedded-1.64.1-x86_64-linux-musl/lib/sass/embedded/varint.rb:19:in `loop'
	from /usr/gem/gems/sass-embedded-1.64.1-x86_64-linux-musl/lib/sass/embedded/varint.rb:19:in `read'
	from /usr/gem/gems/sass-embedded-1.64.1-x86_64-linux-musl/lib/sass/embedded/connection.rb:62:in `block in read'
	from /usr/gem/gems/sass-embedded-1.64.1-x86_64-linux-musl/lib/sass/embedded/connection.rb:61:in `synchronize'
	from /usr/gem/gems/sass-embedded-1.64.1-x86_64-linux-musl/lib/sass/embedded/connection.rb:61:in `read'
	from /usr/gem/gems/sass-embedded-1.64.1-x86_64-linux-musl/lib/sass/embedded/dispatcher.rb:76:in `receive_proto'
	from /usr/gem/gems/sass-embedded-1.64.1-x86_64-linux-musl/lib/sass/embedded/dispatcher.rb:19:in `block (2 levels) in initialize'
	from /usr/gem/gems/sass-embedded-1.64.1-x86_64-linux-musl/lib/sass/embedded/dispatcher.rb:18:in `loop'
	from /usr/gem/gems/sass-embedded-1.64.1-x86_64-linux-musl/lib/sass/embedded/dispatcher.rb:18:in `block in initialize'
Error: Process completed with exit code 1.
```

--- a/css/main.scss
+++ b/css/main.scss
@@ -1,6 +1,3 @@
----
-# Only the main Sass file needs front matter (the dashes are enough)
----
 @charset "utf-8";
 
 /**

